src/NLog/LayoutRenderers/Log4JXmlEventLayoutRenderer.cs

```diff
@@ -75,8 +81,11 @@ public Log4JXmlEventLayoutRenderer() : this(LogFactory.CurrentAppDomain)
         /// </summary>
         public Log4JXmlEventLayoutRenderer(IAppDomain appDomain)
         {
+            _ndcLayoutRenderer = new NdcLayoutRenderer();
```

Comment:
Could the allocation be in the variable declaration since readonly ?

Reply:
Done